samples/buildstrategy/buildkit/buildstrategy_buildkit_cr.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: shipwright.io/v1alpha1
+kind: ClusterBuildStrategy
+metadata:
+  name: buildkit
+  annotations:
+    # See https://github.com/moby/buildkit/blob/master/docs/rootless.md#about---oci-worker-no-process-sandbox for more information
+    container.apparmor.security.beta.kubernetes.io/step-build-and-push: unconfined
+    container.seccomp.security.alpha.kubernetes.io/step-build-and-push: unconfined
```

Comment:
I am not sure if this annotation will still work with Kubernetes 1.19. See https://kubernetes.io/docs/tutorials/clusters/seccomp/#create-a-pod-with-a-seccomp-profile-for-syscall-auditing. Maybe we need to add the securityContext in addition to the `build-and-push` step?

Reply:
Good question. Im running on a v1.19.7 kind node, I can see that my container get its:

```sh
✗ k -n dev get br
NAME                SUCCEEDED   REASON      STARTTIME   COMPLETIONTIME
buildkit-buildrun   True        Succeeded   4m24s       2m54s

✗ k -n dev get pods buildkit-buildrun-l6h9k-pod-8n2hf -o json | jq '.spec.containers[3].securityContext'
{
  "seccompProfile": {
    "type": "Unconfined"
  }
}
```

Reply:
In https://www.suse.com/c/whats-new-in-kubernetes-1-19/ is stated they have 

> A new ‘seccompProfile’ field has been added to the pod and container securityContext objects. (The support for seccomp.security.alpha.kubernetes.io/pod and container.seccomp.security.alpha.kubernetes.io/... annotations are now deprecated, and will be removed in Kubernetes v1.22.0. Right now, an automatic version skew handling will convert the new field into the annotations and vice versa.)

Reply:
I decided to added a comment to remind us and explain on this future deprecation, mainly because:

- users using a k8s older than v1.19, which we currently support will not get the skew handling

Opinions?

Reply:
Looks good.